### DISCUSSION TITLE
botocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the AssumeRole operation

### DISCUSSION THREAD

**Discussion by miogarun** [saleweaver/python-amazon-sp-api#185] [Q&A] [answered]
Hello guys. Pls help me. I'm trying run next code:

`Reports(credentials=credentials).create_report(reportType='GET_MERCHANT_LISTINGS_ALL_DATA')`

But i have this error:

`botocore.exceptions.ClientError: An error occurred (AccessDenied) when calling the AssumeRole operation: User: arn:aws:iam::***:user/*** is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::***:role/***
`

How i can resolve it?

**Comment by miogarun** [comment, marked as answer]
Sorry to bother you, I got it.
You must go to Role - Trust relationships and add user you are using.

**Reply by chaitanyamanani** [reply]
I am facing similar issue. Can you please share what did you do?
